src/platform/plugins/private/vis_types/table/public/components/table_visualization.test.tsx

```diff
@@ -81,7 +81,8 @@ describe('TableVisualizationComponent', () => {
     expect(comp.find('[data-test-subj="tbvChart"]').children().prop('tables')).toEqual([]);
   });
 
-  it('should render split table and set minWidth for column split', () => {
+  // TODO: doesn't work after enabling emotion babel css prop because relies on css prop check
+  it.skip('should render split table and set minWidth for column split', () => {
```

Comment:
this test relies on checking css prop logic and it is no longer available

Reply:
same as https://github.com/elastic/kibana/pull/216489/files#r2028817865

Reply:
thanks,
I am trying: 

```
const splits = comp.find(TableVisSplit).find('div.tbvChart__split'); // div here two help it find

    expect(splits.length).toBe(2);
    splits.forEach((split) => {
      expect(split).toHaveStyleRule('minWidth', `calc(${77 + 22 + 25}px + 2 * 8px)`);
    });
```

and it says 

```
Error: Property not found: minWidth

    at ReactWrapper.toHaveStyleRule (/Users/antondosov/Dev/kibana/src/platform/plugins/private/vis_types/table/public/components/table_visualization.test.tsx:139:21)
```

any idea what might be wrong?

Reply:
because the prop is 'min-width'

Reply:
done